index.html

```diff
@@ -330,22 +335,23 @@
         (depending on the underlying platform).
       </p>
       <ul>
-        <li>User agents MUST NOT allow the website to learn which apps are
-        installed, or which app was chosen from
-        <a><code>navigator.share</code></a>. This information could be used for
-        fingerprinting, as well as leaking details about the user's device.
+        <li>There is a requirement to not allow the website to learn which apps
+        are installed, or which app was chosen from
+        <a><code>navigator.share</code></a>, because this information could be
+        used for fingerprinting, as well as leaking details about the user's
+        device.
         </li>
         <li>Implementors should carefully consider what information is revealed
         in the error message when <a><code>navigator.share</code></a> is
         rejected. Even distinguishing between the case where no targets are
         available and user cancellation may reveal information about which apps
         are installed on the user's device.
         </li>
-        <li>On every call to <a><code>navigator.share</code></a>, the user MUST
-        be presented with a dialog asking them to select a target application
-        (even if there is only one possible target). This surface serves as a
-        security confirmation, ensuring that websites cannot silently send data
-        to native applications.
+        <li>There is a requirement that <a><code>navigator.share</code></a>
+        presents the user with a dialog asking them to select a target
+        application (even if there is only one possible target). This surface
+        serves as a security confirmation, ensuring that websites cannot
+        silently send data to native applications.
         </li>
         <li>Due to the capabilities of the API surface,
         <a><code>navigator.share</code></a> is <a data-cite=
```

Comment:
Done.

"will want to be aware" sounds very watery, but I can't think of anything better that doesn't involve these words.